source/extensions/filters/common/expr/context.cc

```diff
@@ -139,6 +140,20 @@ absl::optional<CelValue> ResponseWrapper::operator[](CelValue key) const {
     return CelValue::CreateMap(&trailers_);
   } else if (value == Flags) {
     return CelValue::CreateInt64(info_.responseFlags());
+  } else if (value == GrpcStatus) {
+    for (const auto wrapper : ((HeadersWrapper[]){trailers_, headers_})) {
```

Comment:
I'm not happy about the linear performance here. Maybe add a TODO to cache the computed value?